bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -181,6 +181,7 @@ spec:
             --cluster-subnets "${OVN_NET_CIDR}" \
             --k8s-service-cidr "${OVN_SVC_CIDR}" \
             --k8s-apiserver "{{.K8S_APISERVER}}" \
+            --ovn-config-namespace openshift-ovn-kubernetes \
```

Comment:
`--ovn-config-namespace` is only relevant if we're using a configmap, which we aren't yet. It seems like this part belongs in #333 and all you need here is the kubectl fix

Reply:
@danwinship it will be used for the master creating endpoints in https://github.com/ovn-org/ovn-kubernetes/pull/763 at some point here too